src/MapfileStyleParser.ts

```diff
@@ -618,7 +618,35 @@ export class MapfileStyleParser implements StyleParser {
   }
 
   /**
-   * Get the GeoStyler-Style Basic Symbolizer Parameter from an Mapfile STYLE / LABEL.
+   * Get the GeoStyler-Style RasterSymbolizer from an Mapfile STYLE.
+   *
+   * @param {object} styleParameters The Mapfile Style Parameters
+   * @return {RasterSymbolizer} The GeoStyler-Style RasterSymbolizer
+   */
+  getRasterSymbolizerFromMapfileStyle(styleParameters: any): RasterSymbolizer {
+    const rasterSymbolizer = { kind: 'Raster' } as RasterSymbolizer;
+
+    if (styleParameters.opacity) {
+      rasterSymbolizer.opacity = parseFloat(styleParameters.opacity) / 100;
```

Comment:
After rebase there is no need to cast with parseFloat any more.

Reply:
OK, fixed both remarks above!